driver/src/main/java/com/code_intelligence/jazzer/junit/JazzerFuzzTestExecutor.java

```diff
@@ -78,47 +78,37 @@ public TestExecutionResult execute() throws IOException, URISyntaxException {
-        // Situation: The user may not be aware of the seed corpus feature.
-        String message = String.format(
-            "Collecting crashing inputs in the project root directory.\nIf you want to keep them organized by "
-                + "fuzz test and automatically run them as regression tests with JUnit Jupiter, create a "
-                + "test resource directory called '%s' in package '%s' and move the files there.",
-            seedCorpusResourcePath, fuzzTestClass.getPackage().getName());
-        request.getEngineExecutionListener().reportingEntryPublished(
-            fuzzTestDescriptor, ReportEntry.from("seed corpus", message));
-      } else {
-        // Situation: The user explicitly configured a seed corpus, but it couldn't be found.
-        throw new FileNotFoundException(
-            String.format("Failed to find seed corpus at '%s' relative to '%s'",
-                fuzzTest.seedCorpus(), fuzzTestClass));
-      }
+      String message = String.format(
+          "Collecting crashing inputs in the project root directory.\nIf you want to keep them organized by "
+              + "fuzz test and automatically run them as regression tests with JUnit Jupiter, create a "
+              + "test resource directory called '%s' in package '%s' and move the files there.",
+          seedCorpusResourcePath, fuzzTestClass.getPackage().getName());
+      request.getEngineExecutionListener().reportingEntryPublished(
+          fuzzTestDescriptor, ReportEntry.from("seed corpus", message));
       libFuzzerArgs.add(String.format("-artifact_prefix=%s%c", baseDir, File.separatorChar));
     } else if ("file".equals(seedCorpusUrl.getProtocol())) {
       // From the second positional argument on, files and directories are used as seeds but not
       // modified. Using seedCorpusUrl.getFile() fails on Windows.
       libFuzzerArgs.add(Paths.get(seedCorpusUrl.toURI()).toString());
       // We try to find the source tree representation of the seed corpus directory and emit
       // findings into it.
-      findSeedCorpusDirectoryInSourceTree().ifPresent(
-          (path)
-              -> libFuzzerArgs.add(
-                  String.format("-artifact_prefix=%s%c", path, File.separatorChar)));
+      seedCorpusSourcePath(fuzzTestClass, baseDir)
+          .ifPresent((path)
+                         -> libFuzzerArgs.add(
+                             String.format("-artifact_prefix=%s%c", path, File.separatorChar)));
     } else {
       // We can't directly use the seed corpus from resources as it's packaged into a JAR. Instead,
       // try to get the path to the seed corpus in the source tree.
-      Optional<Path> seedCorpusSourceDirectory = findSeedCorpusDirectoryInSourceTree();
-      if (seedCorpusSourceDirectory.isPresent()) {
-        libFuzzerArgs.add(seedCorpusSourceDirectory.get().toString());
+      Optional<Path> seedCorpus = seedCorpusSourcePath(fuzzTestClass, baseDir);
+      if (seedCorpus.isPresent()) {
```

Comment:
Should we use the same pattern to check if `seedCorpus` is present as in the other if-branch? So either `seedCorpus.ifPresent()` or `if(seedCorpus.isPresent())`.

Reply:
That case would require `ifPresentOrElse`, which unfortunately isn't available in Java 8. :-(